pkg/util/pod/pod.go

```diff
@@ -90,10 +97,10 @@ func ReadUIntFromLabelBelowBound(obj client.Object, labelKey string, bound int)
 func readUIntFromStringBelowBound(value string, bound int) (*int, error) {
 	uintValue, err := strconv.ParseUint(value, 10, 0)
 	if err != nil {
-		return nil, fmt.Errorf("%w: %s", errInvalidUInt, err.Error())
+		return nil, fmt.Errorf("%w: %s", ErrInvalidUInt, err.Error())
 	}
 	if uintValue > uint64(bound) {
```

Comment:
Can we set here `>=` so it aligns with the function name and the error we return?

Reply:
I already reverted changes. Or what do you mean?

Reply:
My suggestion is like that
```suggestion
	if uintValue >= uint64(bound) {
```

I believe checking if the uintValue is strictly greater is a bug

Reply:
The name of function is below bound. So it's correct, no?

Reply:
Ah, my bad. Thank you so much!